src/Controls/src/Core/BindableObject.cs

```diff
@@ -440,16 +440,17 @@ internal void SetValueCore(BindableProperty property, object value, SetValueFlag
 			}
 			else
 			{
+				var silent = (privateAttributes & SetValuePrivateFlags.Silent) != 0;
 				context.Attributes |= BindableContextAttributes.IsBeingSet;
-				SetValueActual(property, context, value, currentlyApplying, attributes, specificity);
+				SetValueActual(property, context, value, currentlyApplying, attributes, specificity, silent);
```

Comment:
Should we add a unit test for this?

Reply:
Added here: https://github.com/dotnet/maui/pull/16676